go-jni/frontend/handler.go

```diff
@@ -0,0 +1,939 @@
+package frontend
+
```

Comment:
I would propose to name this file go_language_frontend or GoLanguageFrontend, or what the appropriate naming convention for go files is, to relate it to the frontend classes of Java and C++

Reply:
The file contains 1000 LoC so i think for now this is ok.